[nl] Dutch translation

Willem3141 opened the pull request:
# (David, please don't merge this yet :smile: I made this PR for discussing the Dutch translation with Midgard.)

@M1dgard Ik heb een nieuwe branch aangemaakt waar we kunnen werken aan de Nederlandse vertaling.
- [x] Banner
- [x] Titel
- [x] Pagina 1
- [x] Pagina 2
- [x] Pagina 3
- [x] Pagina 4
  - [x] Basisvertaling
  - [x] Beslissen over eerste prentje (#5)
- [x] Pagina 5
- [x] Pagina 6
- [x] Patrons en licentie

Willem3141 committed:
[nl] Start working on the Dutch translation

Only translated the title for now.

M1dgard committed:
[nl] Translate page 1

ghost commented:
Great :)

Willem3141 committed:
[nl] Add initial translation of pages 2-6

Willem3141 commented:
Ik heb een eerste vertaalde versie van pagina 2-6 gecommit :smile:

M1dgard committed:
[nl] Translate patrons and license page

ghost commented:
Nu ook op het laatste blad de standaardvertaling ingevuld.

ghost commented:
Mooi vertaald! Enkel op pagina 4 vind ik het eerste prentje niet zo vlot, maar in het origineel eerlijk gezegd ook niet.

M1dgard committed:
[nl] tenminste in one word

Something the Taalunie and Onze Taal agree on ;)

https://onzetaal.nl/taaladvies/advies/tenminste
http://taaladvies.net/taal/advies/vraag/782/ten_slotte_tenslotte_ten_minste_tenminste_ten_einde_teneinde/

M1dgard committed:
[nl] Edit first panel of page 4

Willem3141 committed:
Merge pull request #5 from M1dgard/dutch

[nl] Suggestion for page 4

Willem3141 commented:
Bedankt voor de verbeteringen! Ik denk dat we nu klaar zijn en kunnen mergen?

ghost commented:
Jep. :smiley:
